noir_stdlib/src/lib.nr

```diff
@@ -107,14 +107,9 @@ where
 
 pub fn wrapping_mul<T>(x: T, y: T) -> T
```

Comment:
Let's deprecate this function

Reply:
Should we have a wrapping trait and have wrapping_add,...wrapping_mul functions in the trait?
It feel a bit strange to handle the multiplication differently (in term of function / trait)

Reply:
I would have separate `WrappingAdd` `WrappingMul`, etc. traits to mirror the arithmetic traits. We can't guarantee that all types which implement one operation will implement the rest.